02 Fx/03 Angular 2/README.md

```diff
@@ -306,7 +352,7 @@ npm install @types/webpack-env --save-dev
 
```

Comment:
In this [section](https://github.com/Lemoncode/webpack-2.x-by-sample/pull/84/files#diff-bd772c9c75aaa0910796cbd1bdf95057R341), it should looks like:

```
  "lib": ["dom", "es2015"],
- "experimentalDecorators":true
+ "experimentalDecorators":true,
+ "types": [
+   "webpack-env"
+ ]
```